.github/workflows/stage-4-acceptance.yaml

```diff
@@ -71,7 +66,7 @@ jobs:
       - uses: actions/upload-artifact@v4
         with:
           name: amplify_outputs.json
-          path: amplify_outputs.json
+          path: frontend/amplify_outputs.json
```

Comment:
More out of curiosity, is there a reason why this path has the filename, but the others don't?

Reply:
i've added the path to the upload-artifact so that when it is downloaded in future jobs it will be installed into the frontend folder - there are other ways of doing it but this works